reactor-core/src/main/java/reactor/core/scheduler/Scheduler.java

```diff
@@ -179,9 +180,8 @@ default void start() {
 	 * Instructs this Scheduler to prepare itself for running tasks
 	 * directly or through its {@link Worker}s.
 	 *
-	 * <p><strong>It should be called only once</strong>. Implementations are free to
-	 * throw an exception if this method is called again either during the runtime of the
-	 * scheduler or after it has been disposed via {@link #dispose()}
+	 * <p>Implementations can throw an exception if this method is called
```

Comment:
what do you think of this alternate wording?
```suggestion
	 * <p>Implementations are encouraged to throw an exception if this method is called
```